benchmarks/data-serving/client/Dockerfile

```diff
@@ -1,24 +1,19 @@
-FROM cloudsuite/java:openjdk8
-
-ENV CASSANDRA_VERSION 3.11.3
+FROM cloudsuite/cassandra:4.0
 
+ENV CASSANDRA_VERSION 4.0-alpha3
 ENV YCSB_VERSION 0.14.0
 
+
 RUN apt-get update \
- && apt-get install gnupg software-properties-common wget -y \
- && buildDeps="python  cassandra-tools=$CASSANDRA_VERSION" \
- && set -x \
- && apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys A278B781FE4B2BDA \
- || apt-key adv --keyserver ipv4.pool.sks-keyservers.net --recv-keys A278B781FE4B2BDA \
- && echo 'deb http://www.apache.org/dist/cassandra/debian 311x main' >> /etc/apt/sources.list.d/cassandra.list \
- && apt-get update \
- && apt-get install -y $buildDeps --no-install-recommends \
- && apt-get autoremove && apt-get clean && apt-get upgrade -y
+    && apt-get install -y wget \
+    &&  wget  -O /tmp/cassandra-tools_${CASSANDRA_VERSION}_all.deb https://downloads.apache.org/cassandra/4.0-alpha3/cassandra-tools_4.0~alpha3_all.deb \
```

Comment:
should use the defined ENV.

Reply:
also aren't these already installed in the base cassandra image?

Reply:
Addressed